fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/_common/util/FoDEnums.java

```diff
@@ -506,4 +506,104 @@ public static AttributeTypes fromInt(int val) {
         }
     }
 
+    public enum VulnerabilitySeverityType {
+        // use same integer values as FoD uses internally
+        Low(1),
+        Medium(2),
+        High(3),
+        Critical(4),
+        Info(-2),
+        BestPractice(-1);
+
+        private final int _val;
```

Comment:
I know similar field & method names are used in other enums in FoDEnums, but can we improve these to make it more clear what values are sent to/received from FoD, for example using field name fodValue and methods getFoDValue and fromFoDValue?